docs/5.0/quickstart.md

```diff
@@ -36,7 +75,32 @@ When setting up Teleport, we recommend running it with Teleports YAML configurat
 $ teleport configure > teleport.yaml
 ```
 
-??? example "Example teleport.yaml" We'll pre-populate your configuration with a nodename and a strong randomly generated static token. yaml --8<-- "snippets/install/teleport-config.yaml"
+!!! tip "Example teleport.yaml"
+
+    ```
+    teleport:
+        nodename: ip-172-31-13-139
+        data_dir: /var/lib/teleport
+        auth_token: f7adb7ccdf04037bcd2b52ec6010fd6f0caec94ba190b765
+    auth_service:
+        enabled: "yes"
+        cluster_name: "teleport-quickstart"
+        listen_addr: 0.0.0.0:3025
+        tokens:
+        - proxy,node,app:f7adb7ccdf04037bcd2b52ec6010fd6f0caec94ba190b765
+    ssh_service:
+        enabled: "yes"
+        labels:
+            env: staging
+    proxy_service:
+        enabled: "yes"
+        listen_addr: 0.0.0.0:3023
+        web_listen_addr: 0.0.0.0:3080
+        tunnel_listen_addr: 0.0.0.0:3024
+        https_keypairs: []
```

Comment:
If this is blank, does it just generate a self-signed cert? (same as not specifying `https_cert_file` and `https_key_file`) @russjones